docker-compose.yml

```diff
@@ -32,3 +32,18 @@ services:
       - grader
     #volumes:
     #  - ./_data/acos/:/var/log/acos
+  radar:
+    image: run-radar
```

Comment:
We will change the image name to `apluslms/run-radar` when we can build the container in Docker Hub in the end.

Reply:
I had missed this, changed now